guides/deploy/starter_deploy_cloudfoundry.html

```diff
@@ -0,0 +1,68 @@
+        <h1 class="header-title">KITURA</h1>
+      </a>
+    </div>
+    <nav class="header-nav">
+      <a class="header-link active-nav" href="../../learn.html">LEARN</a>
+      <a class="header-link" href="../../packages.html">PACKAGES</a>
+      <a class="header-link" href="../../events.html">EVENTS</a>
+      <a class="header-link" href="../../help.html">HELP</a>
+    </nav>
+  </div>
+</header>
+<body>
+  <section class="guide-content">
+    <div class="title-block">
+        <img width="480px" src="../../assets/Kitura.svg" alt="Kitura Logo">
+      <h1>DEPLOY YOUR APP USING CLOUD FOUNDRY</h1>
+    </div>
+
+<p>This guide will show you how to deploy the Kitura application you created by following our <a href="https://www.kitura.io/guides/gettingstarted">Getting Started</a> guide to a <a href="#cloud-foundry">Cloud Foundry</a> based cloud environment. The guide assumes you already have an environment to deploy to. If not, you can <a href="https://console.bluemix.net/registration/">sign up</a> to IBM Cloud for free and use it to host your Cloud Foundry apps.</p>
+<h4 >Starter project contents</h4>
+<p>Your generated Kitura project already contains a manifest.yml file which you can use to deploy your application. This file is optional, but allows you to maintain consistency across deployments. If necessary you can adjust or add attributes, a list of which can be found <a href="https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#optional-attributes">here</a>. You’ll use the Cloud Foundry command line interface, which you can install using the instructions <a href="https://github.com/cloudfoundry/cli#downloads">here</a>.</p>
+<h4 >Deploy to Cloud Foundry</h4>
+<p>To push your application, you first need to consult your cloud operator to find the URL you need to target. You can set it in the command line like this:</p>
+<div><pre><code class="sourceCode bash">cf login -a &lt;API endpoint&gt;</code></pre></div>
+<p>You’ll also need to provide your username and password when prompted. Next, you’ll need to find the organisation and space you are pushing to, and target them like this:</p>
+<div><pre><code class="sourceCode bash">cf target -o &lt;organisation&gt; -s &lt;space&gt;</code></pre></div>
+<p>Now, you’re ready to push your app. Simply run the following command from the same location as your manifest.yml:</p>
+<div><pre><code class="sourceCode bash">cf push</code></pre></div>
+<p>Your Kitura application should now be successfully deployed into your Cloud Foundry environment! If you haven’t edited the <code>manifest.yml</code> file, it will have been assigned a random route. If you visit this URL you should see the Kitura splash page.</p>
+
+<section class="slack-help">
+  <a href="http://slack.kitura.io/">
+    <img width="80px" src="../../assets/slack-icon.png" alt="Slack icon">
+    <h2>NEED HELP?</h2>
+    <h2>MESSAGE US ON SLACK.</h2>
+  </a>
+</section>
+</body>
+<footer>
+
+<nav class="footer-nav">
```

Comment:
David tells me that the footer is in the wrong place in this file.

Reply:
Correct, the footer in this guide is incorrect. It's sitting inside the white section of the guide rather than the foot of the page. 

After looking at the code, it's not that the footer is in the wrong place it seems you've missed a closing `</section>` tag. 

If you add a `</section>` tag just before the `</body>` tag it should resolve this issue :).